docker/Dockerfile

```diff
@@ -55,17 +88,9 @@ ENV ESMF_BOPT=O3
 RUN git clone -b ESMF_8_0_0 --depth 1 https://git.code.sf.net/p/esmf/esmf $ESMF_DIR
 RUN cd $ESMF_DIR && make lib -j8 && make install && make installcheck
 
-## Build FV3 executable in its own image
+## Copy FV3GFS sources for non-serialize image
 ##---------------------------------------------------------------------------------
-FROM fv3gfs-environment AS fv3gfs-build
-
-ENV FMS_DIR=/FMS
-ENV ESMF_DIR=/usr/local/esmf
-ENV ESMF_INC="-I/usr/local/esmf/include -I${ESMF_DIR}/mod/modO3/Linux.gfortran.64.mpiuni.default/"
-ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${ESMF_DIR}/lib/libO3/Linux.gfortran.64.mpiuni.default/:${FMS_DIR}/libFMS/.libs/
-
-COPY --from=fv3gfs-fms /FMS $FMS_DIR
-COPY --from=fv3gfs-esmf /usr/local/esmf ${ESMF_DIR}
+FROM ubuntu:19.10 AS fv3gfs-sources-false
```

Comment:
'sources' is a little confusing (though you DO have a comment about it, so that helps) -- any reason to not call it fv3gfs-serialize-false?

Reply:
It's used regardless of whether you serialize the model. Would `fv3gfs-src` be clearer? I thought I'd expand `src` to `sources` to be more readable.

Reply:
Oh, are you suggesting we make `fv3gfs-serialize-true` and `fv3gfs-serialize-false` and then say `FROM fv3gfs-serialize-$serialze AS fv3gfs-sources`? That could work nicely actually, I'll do that..

Reply:
yeah that's what I meant -- thanks!

Reply:
Done. Used `fv3gfs-src-serialize-$serialize`.